tiledbimg/converters/base.py

```diff
@@ -15,15 +16,31 @@ def level_count(self) -> int:
 
     @abstractmethod
     def level_image(self, level: int) -> np.ndarray:
-        """Return the image for the given level as (x, y, RGB) 3D numpy array"""
+        """Return the image for the given level as (X, Y, C) 3D numpy array"""
 
 
-class ImageConverter(ABC):
-    _rgb_dtype = np.dtype([("", "uint8"), ("", "uint8"), ("", "uint8")])
+@dataclass(frozen=True)
+class Dimension:
+    name: str
+    max_tile: int
+
+    def to_tiledb(self, size: int) -> tiledb.Dim:
```

Comment:
`to_tiledb_dim`?

Reply:
I thought It's implied since `self` is a dimension, can rename it if not obvious.

Reply:
For sure it's obvious, so your call. It's just an intuition of mine that `to_tiledb` = `to_tiledb_array`.